README.md

```diff
@@ -163,7 +163,7 @@ and <a href="https://js.tensorflow.org/api/latest/">documentation</a> for more d
 ## Importing pre-trained models
 
 We support porting pre-trained models from:
-- [TensorFlow SavedModel](https://github.com/tensorflow/tfjs-converter)
+- [TensorFlow SavedModel](https://www.tensorflow.org/js/tutorials/conversion/import_saved_model)
```

Comment:
I think this doc is more complete and updated: https://github.com/tensorflow/tfjs/tree/master/tfjs-converter#getting-started.